requirements.tex

```diff
@@ -143,62 +145,64 @@ \subsection{Stakeholders Requirements} \label{sec:stakeholdersreqs}
 The following list of requirements is derived from the above section \ref{sec:reqs}.
 
 
-\subsubsection{Releases Schedule Requirement} \label{sec:milestone}
+\subsubsection{Release Schedule Requirement} \label{sec:milestone}
 
 Releases on a software product shall be scheduled in advance.
 
 Two types of release schedule can be identified:
 
 \begin{itemize}
-\item Functional Based Release Schedule: a release shall provide a requested functionality.
+\item Functional Based Release Schedule: a release shall provide specific functionality.
 \item Time-Based Release Schedule: a release shall be provided on a specific date or cadence.
 \end{itemize}
 
-In both cases, releases can be tight to project milestones.
-Additional releases can be requested to the DM-CCB using the RFC mechanism.
+In both cases, releases may be associated with project milestones.
+Additional releases may be made available upon request to the DM-CCB using the “RFC” mechanism as described in \citeds{DMTN-106}.
```

Comment:
The RFC approach is described in this document, section 6. DMTN-106 may give technicalities on how to do path releases, and DevGuide on how to backport, but how to request a new release via RFC is described here.

Reply:
ok